netlify.toml

```diff
@@ -12,7 +12,7 @@
     X-XSS-Protection = "1; mode=block"
     X-Content-Type-Options = "nosniff"
     Referrer-Policy = "same-origin"
-    Content-Security-Policy = "default-src 'self'; connect-src 'self' analytics.google.com stats.g.doubleclick.net; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com analytics.google.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
+    Content-Security-Policy = "default-src 'self'; connect-src 'self' analytics.google.com stats.g.doubleclick.net res.cloudinary.com; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com analytics.google.com; img-src 'self' blob: data: https:; frame-ancestors 'none';"
     Permissions-Policy = "camera=(), microphone=(), geolocation=(), interest-cohort=()"
 
 [[headers]]
```

Comment:
I just remember, we need to make sure `/service-worker.js` never send any cache header.
I think we need to add new header configuration and set it  as `no-store, max-age=0`